test/index.js

```diff
@@ -617,6 +617,14 @@ describe('onBuild()', () => {
   test('does not generate an ipx edge function if Netlify Edge is disabled', async () => {
     process.env.NEXT_DISABLE_NETLIFY_EDGE = '1'
     await moveNextDist()
+
+    // We need to pretend there's no edge API routes, because otherwise it'll fail
+    // when we try to disable edge runtime.
+    const manifest = path.join('.next', 'server', 'middleware-manifest.json')
+    const manifestContent = await readJson(manifest)
+    manifestContent.functions = {}
+    await writeJSON(manifest, manifestContent)
```

Comment:
This doesn't block the PR, but could we not have a separate test for the scenario when `NEXT_DISABLE_NETLIFY_EDGE=1` and we have routes, that we expect it to throw? Or maybe I'm misunderstanding the error being explained here.